src/form/Form.jsx

```diff
@@ -210,6 +211,11 @@ function Form({
   useEffect(() => {
     if (!activeStep) return;
 
+    Object.entries(initialValues).forEach(([key, val]) => {
```

Comment:
i think this logic should be flipped. we run through activeStep and look for any signature fields. for those fields, see if there exists a value for it. if so, call fromDataURL and initialize it